.coveragerc

```diff
@@ -1,5 +1,9 @@
 [report]
-omit = */samples/*
+omit =
+    */samples/*
+    # Don't report coverage over platform-specific modules.
+    oauth2client/contrib/_fcntl_opener.py
```

Comment:
Ah OK. No worries then. OK to merge? (Also do you have a timeline for `multistore` getting 100% and when you're swapping out for fasteners?)